modules/ROOT/pages/users.adoc

```diff
@@ -86,6 +86,16 @@ include::partial$include-nav-users.adoc[]
 
 The user now has the permissions you selected in the business groups and environments you specified.
 
+== Managing Deleted Users
+
+When you delete a user from your root organization, the user is _soft deleted_, wherein a user can no longer access the root organization or business groups and becomes a dangling Anypoint Platform account unless associated with another organization. Anypoint Platform maintains a record of the user's presence for the purposes of audit logs. 
```

Comment:
I'm really hung up on dangling here because it seems like such a strange word to use, but any edits I came up with just made it longer. Should you do "dangling"?

Reply:
When you delete a user from your root organization, the user is _soft deleted_. This user can no longer access the root organization or business groups but maintains an Anypoint Platform account and can later be associated with another organization. Anypoint Platform maintains a record of such users for the purposes of audit logs.

Reply:
```suggestion
When you delete a user from your root organization, the user is _soft deleted_. This user can no longer access the root organization or business groups but maintains an Anypoint Platform account and can later be associated with another organization. Anypoint Platform maintains a record of such users for the purposes of audit logs.
```